src/components/Header/Header.js

```diff
@@ -0,0 +1,18 @@
+import './Header.css';
+
+import headerLogo from './header-logo.png';
+import stars from './stars.png';
+
+function Header() {
+  return (
+    <div className="header">
+      <img className="stars" src={stars} alt="stars" />
+      <a className="header__logo">
```

Comment:
Why do you need a link here?

Reply:
The logo is a link for the main page. It will be changed in the following tasks.